DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/DependencyInjection/AutofacArcGISModule.cs

```diff
@@ -34,6 +34,8 @@ protected override void Load(ContainerBuilder builder)
     builder.RegisterType<ArcGISDocumentStore>().SingleInstance();
 
     // Register bindings
+    builder.RegisterType<TestBinding>().As<IBinding>().SingleInstance();
+    builder.RegisterType<ConfigBinding>().As<IBinding>().SingleInstance().WithParameter("connectorName", "ArcGIS"); // POC: Easier like this for now, should be cleaned up later
```

Comment:
I don't have too strong feeling about this, but arguable there may be a ArcGISContext that already has this string or could have it... certainly can live for a bit with the POC comment